### PR TITLE
Update Travis config to fix cert errors

### PR DESCRIPTION
**Pull request by WillMoggridge**
Update requests and install patch used on www.ubuntu.com's config

## QA

Tests pass and link checker doesn't have HTTPS errors

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,7 +15,7 @@ install:
   - pip install flake8==3.0.4
   - pip install -r requirements.txt
   - sudo apt-get update
-  - sudo apt-get install -y linkchecker
+  - sudo apt-get install -y linkchecker libssl1.0.0
```
Comment (tbille) [inline .travis.yml]: useless: `libssl1.0.0 is already the newest version (1.0.1f-1ubuntu2.22).`

Comment (tbille) [reply .travis.yml]: https://travis-ci.org/canonical-websites/developer.ubuntu.com/builds/294942510#L954

**Comment by tbille** [comment]
please squash all commits to get one commit :)

**Comment by WillMoggridge** [comment]
Give me a moment!

**Comment by WillMoggridge** [comment]
This is passing now, and rebasing should help with #322 #323 #324

**Review by tbille — APPROVED** [review]

**Pull request merged**